4.1.2 breaks our build  with " error TS1086: An accessor cannot be declared in an ambient context."

We do not use gtoken directly and this started happening just after 4.1.2 rolled out. It seems to be a very similar issue we reported [here](https://github.com/googleapis/gax-nodejs/issues/642) for google-gax.

```
.../node_modules/gtoken/build/src/index.d.ts:32:9 - error TS1086: An accessor cannot be declared in an ambient context.

32     get accessToken(): string | undefined;
           ~~~~~~~~~~~

.../node_modules/gtoken/build/src/index.d.ts:33:9 - error TS1086: An accessor cannot be declared in an ambient context.

33     get idToken(): string | undefined;
           ~~~~~~~

.../node_modules/gtoken/build/src/index.d.ts:34:9 - error TS1086: An accessor cannot be declared in an ambient context.

34     get tokenType(): string | undefined;
           ~~~~~~~~~

.../node_modules/gtoken/build/src/index.d.ts:35:9 - error TS1086: An accessor cannot be declared in an ambient context.

35     get refreshToken(): string | undefined;
           ~~~~~~~~~~~~

```

Our environment:
```
$ node --version
v10.16.2
$ yarn --version
1.17.3
$ tsc --version
Version 3.5.3
```

Workaround:
In package.json add:
```
"resolutions": {
    "gtoken": "4.1.1"
  }
```

--- a/package.json
+++ b/package.json
@@ -57,7 +57,7 @@
     "nock": "^11.3.2",
     "nyc": "^14.0.0",
     "source-map-support": "^0.5.6",
-    "typescript": "~3.7.0",
+    "typescript": "~3.6.0",
     "linkinator": "^1.5.0"
   },
   "files": [

